develop/tutorials/articles/260-product-navigation/00-intro.markdown

```diff
@@ -0,0 +1,28 @@
+# Product Navigation [](id=product-navigation)
+
+Liferay's product navigation consists of the main menus you use to customize,
+configure, and navigate your Liferay instance. Whether you edit a page, switch
+to a different site scope, access a user's credentials, etc., you're constantly
+using the default navigation menus. Liferay's product navigation is designed to
+be intuitive and extensive, but often times, providing a customization to a
+default menu is necessary to give your Liferay instance that unique touch you're
+searching for. Liferay allows developers to extend and customize the default
+product navigation to fit their needs.
+
```

Comment:
Comment test.

Reply:
Reply test

Reply:
In any case, here I think it's always work, the problem was with the pull to himself.

The problem with using my repo is that it's very out of date and I'm running into problems with GitHub that prevent me from updating it.

Reply:
I've had that happen to me before with liferay-portal. In that case, I nuke it from orbit: rm -rf my local copy, delete my GitHub fork, re-fork, and re-clone. It's the only way to be sure. :-)